src/components/Configuration/components.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Popover from 'react-tiny-popover';
+import './Configuration.scss';
+
+const isFirstChild = child => typeof child === 'string' && child !== ' ';
+
+const removeSpaces = child => (isFirstChild(child) ? child.trim() : child);
+
+const addLink = (child, i, url) => {
+  return isFirstChild(child) ? (
+    <a href={url} key={i}>
+      {child}
+    </a>
+  ) : (
+    child
+  );
+};
+
+const Card = ({ body }) => {
+  return (
+    <div className="markdown">
+      <pre className="inline">
+        <code>{body}</code>
+      </pre>
+    </div>
+  );
+};
+
+export class Modes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      summary: null,
+      content: null
+    };
+  }
+
+  componentDidMount() {
+    const { children, url } = this.props;
+
+    const closeDefault = children.findIndex(child => {
+      if (React.isValidElement(child)) {
+        return child.props.props.className.includes('tag') && child.props.children.length === 4;
```

Comment:
how did we get `.props.props`? maybe worth renaming the latter if its not reacts `props`

Reply:
I don't think so, that is the convention with remark ecosystem.
Will check